.golangci.yml

```diff
@@ -63,3 +63,7 @@ linters-settings:
         disabled: true
   lll:
     line-length: 120
+  tagliatelle:
+    case:
+      rules:
+        json: "snake"
```

Comment:
@xinaxu  I had to change this linter rule, because the struct I'm getting from Ribas side is using snake case for the json (ex `provider_id` not `providerId`). So, we need this to match up for the unmarshaling to work. 

The default config is to use `camelCase` here. 

I noticed in Singularity it's just using the default so it's `camelCase` - do you have any strong opinion on this (i.e, should we change the json format instead to maintain consistency)?

Reply:
hmm, alternatively we could just disable the `tagliatelle` linter altogether but I think it's better to have this as a nice check on the struct tags

Reply:
If it is the only place where json field is snake case, then that's fine. If in the future we encounter other files that use other case style, then we can add //nolint:tagliatelle to the struct